autoinstallation/bin/agama-auto

```diff
@@ -4,23 +4,30 @@ set -ex
 # Temporarily skip the AutoYaST XML validation
 export YAST_SKIP_XML_VALIDATION=1
 
-if [ -z "$1" ]
-then
-  url=$( grep 'agama.auto=' < /run/agama/cmdline.d/agama.conf | awk -F 'agama.auto=' '{sub(/ .*$/, "", $2); print $2}' )
+if [ -z "$1" ]; then
+	url=$(grep 'agama.auto=' </run/agama/cmdline.d/agama.conf | awk -F 'agama.auto=' '{sub(/ .*$/, "", $2); print $2}')
 else
-  url="$1"
+	url="$1"
 fi
 
-if [ -z "$url" ]
-then
-  echo "no autoinstallation profile"
-  exit 0
+if [ -z "$url" ]; then
+	echo "no autoinstallation profile"
+	exit 0
 fi
 
+method=$(awk -F 'agama.finish=' '{sub(/ .*$/, "", $2); print $2}' </run/agama/cmdline.d/agama.conf)
+
 echo "Using the profile at $url"
 
 case "$url" in
-* )
-    /usr/bin/agama profile import "$url"
-    /usr/bin/agama install;;
+*)
+	/usr/bin/agama profile import "$url"
+	/usr/bin/agama install
+	case "$method" in
+	"stop") ;;
```

Comment:
maybe you can log it with simple echo call.

Reply:
agama finish stop is already supported. so adapted it